.github/workflows/ios.yml

```diff
@@ -7,16 +7,32 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
-    name: Build and Test on iOS
-    runs-on: macos-latest
+  build-and-test-iOS:
+    name: Build and Test on iOS ${{ matrix.iOS-version }}
+    runs-on: macos-11
+    strategy:
+      fail-fast: false
```

Comment:
with fail-fast on, if one generated task fails the rest will all be aborted. this didn't seem very useful since it would make it hard to tell if something was only failing on one iOS version or all.